diff --git a/template/en/default/list/table.html.tmpl b/template/en/default/list/table.html.tmpl
index 282ea51..15fd33c 100644
--- a/template/en/default/list/table.html.tmpl
+++ b/template/en/default/list/table.html.tmpl
@@ -71,6 +71,8 @@
 
 [% PROCESS bug/time.html.tmpl %]
 
+[% Hook.process("before_editable_fields") %]
+
 [%############################################################################%]
 [%# Table Header                                                             #%]
 [%############################################################################%]
@@ -193,6 +195,8 @@
     [% FOREACH column = displaycolumns %]
     <td [% 'style="white-space: nowrap"' IF NOT abbrev.$column.wrap %]
         class="bz_[% column FILTER css_class_quote %]_column">
+[% Hook.process('field_start') %]
+
       [% IF abbrev.$column.maxlength %]
         <span title="[%- display_value(column, bug.$column) FILTER html %]">
       [% END %]
@@ -214,6 +218,7 @@
       [% IF abbrev.$column.maxlength %]
         </span>
       [% END %]
+[% Hook.process('field_end') %]
     </td>
     [% END %]
 
